README.md

```diff
@@ -3,50 +3,62 @@
-   1. Linux VCPKG (G++ 7 or CLang 4)
-      1. `cmake -DCMAKE_TOOLCHAIN_FILE=**VCPKG_DIR**/scripts/buildsystems/vcpkg.cmake ..` in the build directory
-      1. `make` in the build directory
-   1. Windows (Visual Studio 2019)
-      1. Run cmake-gui setting the source line to the root directory and the build line to the build directory.
-      1. Hit configure and select `Specify toolchain file for cross-compiling` using `**VCPKG_DIR**/scripts/buildsystems/vcpkg.cmake`.
-      1. Click generate; then open and build the solution in Visual Studio.
-      1. In the project properties for `trillek-client` change the `Debugging`->`Working Directory` to `$(SolutionDir)..\`.
-      1. **Potentially** Download and install oalinst.zip (OpenAL installer) http://openal.org/downloads/ and install it.
-   1. MaxOS X 10.15 (10.14 should work, following the instructions above)
-      1. `brew install bullet`
-      1. `cmake -DCMAKE_TOOLCHAIN_FILE=**VCPKG_DIR**/scripts/buildsystems/vcpkg.cmake ..` in the build directory
-      1. `make` in the build directory
-1. Run it from `tec/`
+#### Windows (Visual Studio 2019)
+   1. Run cmake-gui setting the source line to the root directory and the build line to the build directory.
+   1. Hit configure and select `Specify toolchain file for cross-compiling` using `**VCPKG_DIR**/scripts/buildsystems/vcpkg.cmake`.
+   1. Click generate; then open and build the solution in Visual Studio.
+   1. In the project properties for `trillek-client` change the `Debugging`->`Working Directory` to `$(SolutionDir)..\`.
+   1. **Potentially** Download and install oalinst.zip [OpenAL installer](http://openal.org/downloads/) and install it.
+#### Linux
+##### VCPKG (G++ 7 or CLang 4)
+   1. `cmake -DCMAKE_TOOLCHAIN_FILE=**VCPKG_DIR**/scripts/buildsystems/vcpkg.cmake ..` in the build directory
+   1. `make` in the build directory
+##### NON-VCPKG - G++ 7 or CLang 4 (Unsupported due to cmake)
+   1. **INSTALL REQUIRED LIBS** bullet, glew, glfw3, glm, asio, lua, openal-soft, spdlog, Dear ImGui, sol3(also known as sol2), and protobuf. Some of these will need versions not in your distribution (just ask for help in the IRC or Discord.)
+      1. If you are on Ubuntu/Debian/etc. (something with `apt`):
+         1. Run `apt-get install libglew-dev libglfw3 libglm-dev libasio-dev`
+         2. Run `apt-get install liblua5.2-dev libopenal-dev  libbullet-dev`
+         3. Run `apt-get install libprotobuf-dev protobuf-compiler libspdlog-dev`
+      1. If you are on Arch/etc. (something with `pacman`):
+         1. Run `pacman -S glew glfw-x11 glm asio lua52 openal bullet protobuf spdlog`
+   1. `cmake ..` in the build directory
+   1. `make tec` in the build directory
 
-### Unit tests
+#### Mac OS (10.15 Recommended, 10.14 Should work)
+   1. `brew install bullet`
+   1. `cmake -DCMAKE_TOOLCHAIN_FILE=**VCPKG_DIR**/scripts/buildsystems/vcpkg.cmake ..` in the build directory
+   1. `make` in the build directory
+   1. Run it from `tec/`
```

Comment:
Just noticed, this step. We should maybe just remove it as it applies to all platforms, but it is also kind of misleading.